unime/src/routes/+layout.svelte

```diff
@@ -57,46 +59,121 @@
       goto(`/prompt/${type}`);
     }
   }
+
+  interface DevModeButton {
+    stringIcon?: string;
+    svelteIcon?: typeof SvelteComponent<SvelteHTMLElements['svg']>;
+    onClick: () => void;
+  }
+
+  function createDevButtons(): DevModeButton[] {
+    const backBtn: DevModeButton = {
+      svelteIcon: ArrowLeft,
+      onClick: () => history.back(),
+    };
+
+    const resetBtn: DevModeButton = {
+      svelteIcon: Trash,
+      onClick: () => dispatch({ type: '[App] Reset' }),
+    };
+
+    const ferrisBtn: DevModeButton = {
+      stringIcon: '🦀',
+      onClick: () => loadFerrisProfile(),
+    };
+
+    const dragonBtn: DevModeButton = {
+      stringIcon: '🐲',
+      onClick: () => (showDragonProfileSteps = !showDragonProfileSteps),
+    };
+
+    const debugBtn: DevModeButton = {
+      svelteIcon: ScrollText,
+      onClick: () => (showDebugMessages = !showDebugMessages),
+    };
+
+    return [backBtn, resetBtn, debugBtn, ferrisBtn, dragonBtn];
+  }
+
+  const devButtons = createDevButtons();
+
+  // Order needs to match the BE: 'ProfileSteps' enum, it needs to be the same order because every step is based upon the previous.
+  // 'AddCredentials' is ran after 'CreateProfile' and 'AcceptCredentials' after 'AddCredentials', etc.
+  const profileSteps: ProfileSteps[] = [
+    'CreateProfile',
+    'AddCredentials',
+    'AcceptCredentials',
+    'AddConnection',
+    'AcceptConnection',
+    'AddPresentation',
+    'ShareCredentails',
+    'AddFutureEngineer',
+    'CompleteFlow',
+  ];
+
+  async function loadFerrisProfile() {
+    dispatch({ type: '[DEV] Load DEV profile', payload: { profile: 'Ferris', execute_steps: null } }).then(() => {
+      // Reload page
+      setTimeout(async () => {
+        await goto('/');
+        await goto('/me');
+      }, 500);
```

Comment:
Can you give a little more info why you chose a timer that executes two `goto()`'s instead of something native such as `window.location.reload()`?